### PR TITLE
vsup_ifis edited to use getHashedData

### PR DESCRIPTION
**Pull request by balcirakpeter**
- For this purpose was used method getHashedHierarchicalData.

### CODE REVIEW

File: gen/vsup_ifis [hunk 1]
```diff
@@ -65,35 +65,43 @@ my $users;  # $users->{$uco}->{ATTR} = $attrValue;
...
+		my $firstName = $data->getUserAttributeValue(member => $memberId, attrName => $A_FIRST_NAME);
+		my $artisticLastName = $data->getUserAttributeValue(member => $memberId, attrName => $A_ARTISTIC_LAST_NAME);
+		my $lastName = $data->getUserAttributeValue(member => $memberId, attrName => $A_LAST_NAME);
+		my $titleAfter = $data->getUserAttributeValue(member => $memberId, attrName => $A_TITLE_AFTER);
+		my $birthNumber = $data->getUserAttributeValue(member => $memberId, attrName => $A_BIRTH_NUMBER);
+		my $gender = $data->getUserAttributeValue(member => $memberId, attrName => $A_GENDER);
+		my $job = $data->getUserAttributeValue(member => $memberId, attrName => $A_JOB);
+		my $prefMail = $data->getUserAttributeValue(member => $memberId, attrName => $A_VSUP_PREF_MAIL);
+		my $vsupMail = $data->getUserAttributeValue(member => $memberId, attrName => $A_VSUP_MAIL);
+		my $userLogin = $data->getUserFacilityAttributeValue(member => $memberId, attrName => $A_LOGIN);
```
Comment (zlamalp) [inline gen/vsup_ifis:91]: We use single login-namespace in this service, so the attribute ``login`` is of ``user`` type and not ``user_facility``.

File: gen/vsup_ifis [hunk 2]
```diff
@@ -133,15 +156,18 @@ foreach my $rData ($data->getChildElements) {
...
+		if ($bankAccountDC2) {
+			$users->{$key}->{'BANK_ACCOUNT'} = $bankAccountDC2;
+		} elsif ($bankAccountKOS) {
+			$users->{$key}->{'BANK_ACCOUNT'} = $bankAccountKOS;
 		} else {
 			$users->{$key}->{'BANK_ACCOUNT'} = '';
 		}
 
-		$users->{$key}->{$A_USER_ID} = $uAttributes{$A_USER_ID};
+		$users->{$key}->{$A_USER_ID} = $data->getUserAttributeValue(member => $memberId, attrName => $A_USER_ID);
```
Comment (zlamalp) [inline gen/vsup_ifis:170]: We can replace this with ``getUserIdForMember(member => $memberId)`` and remove ```user:core:id``` attribute from the required attributes of the service.

**Review by zlamalp — CHANGES_REQUESTED** [review]
Please update also script version. Thank you.

File: gen/vsup_ifis (hunk 1 above)
Comment (balcirakpeter) [reply gen/vsup_ifis]: Fixed

File: gen/vsup_ifis (hunk 2 above)
Comment (balcirakpeter) [reply gen/vsup_ifis]: Fixed

**Review by zlamalp — APPROVED** [review]

**Pull request merged**